### ISSUE TITLE
Developer plugins

### ISSUE THREAD

**Issue by armandzwan** [AvdLee/RocketSimApp#317]
**Is your feature request related to a problem? Please describe.**
No, it's not. I just had a brainwave.

**Describe the solution you'd like**
Wouldn't it be great if we (all) would have the opportunity to enhance RocketSIM bij developing plugins for it?

**Describe alternatives you've considered**
I don't see any alternative but hacking RocketSIM.

**Additional context**
I came up with this idea, because I saw the latest update with "NEW DEVELOPER UTILITY" and I thought someone had released a plugin for RocketSIM.

**Comment by AvdLee** [comment]
Yes, totally! This is on my private backlog actually, but not an easy thing to build. Thanks for posting your thoughts, I'll see what I can do!
